src/templates/ratings/add.html

```diff
@@ -7,6 +7,12 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="text-small">
+           {{ _('Please read the <a href="https://developer.mozilla.org/en-US/Marketplace/User_Review_Guidelines">Review Guidelines</a> for more details about rating apps.
```

Comment:
I thought there was code in [fireplace](https://github.com/mozilla/fireplace) / [marketplace-core-modules](https://github.com/mozilla/marketplace-core-modules) for `$('[rel=external]').attr('target', '_blank')`. I guess not.

so, yes, just make it `target="_blank"`.